app/helpers/talks_helper.rb

```diff
@@ -1,2 +1,21 @@
 module TalksHelper
+  def link_to_talk(talk = nil, options = {}, html_options = {}, &block)
+    raise ArgumentError, "must pass an instance of talk to this helper method" unless talk.present? && talk.is_a?(Talk)
+
+    url, target = if talk.date > 2.weeks.ago
```

Comment:
The problem with this is that this date of the talk is meant the date when the talk was held, not when it was released.

Reply:
ahhh I missed that. 🤔 I guess we would have to expand the data/inject model to track this information.

Reply:
Yeah sorry, I guess that's what I was alluding to in https://github.com/adrienpoly/rubyvideo/pull/229#issuecomment-2419259464

Reply:
We could get the information from youtube when users add more videos, and run a script to backfill the data model. 

If it will allow rubyvideo to have the latest videos, I think it's worth it.